src/style.css

```diff
@@ -1,3 +1,85 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
```

Comment:
Don't use the `*` selector as it impacts performance. Set styles only for elements that require them. Consider setting `margin`, `padding`, and `box-sizing` in a more specific selector or element.